firmware/examples/Uart/source/main.cpp

```diff
@@ -8,30 +8,36 @@
 
 int main(void)
 {
-    // This example code uses a loopback test Please connect the relevant TxD
-    // and RxD pins together. For this example please connect the TxD2 and RxD2
-    // pins together.
-    Uart uart2(Uart::Channels::kUart2);
-    // Calls the initialize function and checks if the
-    // function fails due to wrong input.
-    uart2.Initialize(38400);
-    // While loop to constantly send the array of data
-    while (true)
+  DEBUG_PRINT("Uart application starting.");
```

Comment:
```suggestion
  LOG_INFO("Uart application starting.");
```
Please use LOG_INFO, rather than DEBUG_PRINT. I will be deprecating DEBUG_PRINT soon.